src/NuGet.Services.V3Repository/V3PathCalculator.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace NuGet.Services.V3Repository
+{
+    public class V3PathCalculator

```

Comment:
Can this guy be named `V3PathResolver` or `V3PathGenerator` or something like that? Calculator is not super descriptive.